backend/spvm.pm

```diff
@@ -22,6 +22,7 @@ use base 'backend::virt';
 
 use testapi qw(get_var get_required_var check_var);
 use IO::Select;
+use backend::svirt;
```

Comment:
IMHO this is not needed (you use fully qualified name `backend::svirt::get_ssh_output($chan)`, not just `get_ssh_output($chan)`.